app/controllers/referee_interface/reference_controller.rb

```diff
@@ -23,6 +23,8 @@ def submit_feedback
       )
 
       if @reference_form.save
+        SendReferenceConfirmationEmail.call(application_form: @application, reference: reference)
```

Comment:
The feature flag could go here as well; currently, it’s not clear that if it’s off, this service does a noop

Reply:
That's a fair point! Will move it here.